addons/mail/static/src/discuss/core/web/discuss_core_web_service.js

```diff
@@ -51,6 +52,9 @@ export class DiscussCoreWeb {
                 "%(user)s connected. This is their first connection. Wish them luck.",
                 { user: username }
             );
+            if (!this.multiTab.isOnMainTab()) {
+                return;
+            }
             this.notificationService.add(notification, { type: "info" });
             const chat = await this.store.getChat({ partnerId });
```

Comment:
I would early return after showing the notification, so all tabs show the reason this chat window is open.

Note that the problem may also work with multiple devices, but this is far less frequent and could be improved later when this becomes a higher priority.